yt/frontends/openPMD/data_structures.py

```diff
@@ -196,12 +196,15 @@ def _parse_index(self):
             0] = self.dataset.domain_right_edge  # (N, 3) <= float64
         self.grid_dimensions[
             0] = self.dataset.domain_dimensions  # (N, 3) <= int
-        self.grid_particle_count[
-            0] = self.dataset._handle[
-                self.basePath +
-                particlesPath +
-                "/electrons/position/x"].shape[
-            0]  # (N, 1) <= int
+# TODO this disables particle reads for now
+#      Should might be set in _read_particles for each species,
+#      also each species might need its own grid (?)
+#        self.grid_particle_count[
```

Comment:
idea one:
- select the first species for now and only work with that.

generic implementation:
- looks like we need *one grid per species* (is that correct? ask list)